manifests/kibana.pp

```diff
@@ -6,7 +6,7 @@
   $kibana_version = '7.2.0',
   $kibana_app_version = '3.9.4_7.2.0',
 
-  $kibana_elasticsearch_ip = 'localhost',
+  $kibana_elasticsearch_ip = '<Put Elasticsearch IP>',
```

Comment:
Rename to `<YOUR_ELASTICSEARCH_IP>` please

Reply:
Hi Jose!

Thanks for your attention. Check the changes [here](https://github.com/wazuh/wazuh-puppet/commit/1301d408b350d150f40ea1e58dbace59119cc99a).

Kind regards,

Rshad